support/win_prng_support/src/lib.rs

```diff
@@ -25,19 +25,25 @@ macro_rules! use_win10_prng_apis {
         $($crate::use_win10_prng_apis!(@x $lib);)*
     };
     (@x advapi32) => {
+        // SAFETY: We are purposefully overriding this symbol and we have made
```

Comment:
instead of copy-pasting the same blurb across N > 3 uses of no_mangle, I would have a single mod-level blurb explaining the safety, and then having these safety comments simply be `// SAFETY: see module level safety justification`